app/grants/views.py

```diff
@@ -291,6 +291,7 @@ def grant_fund(request, grant_id):
         'active': 'fund_grant',
         'title': _('Fund Grant'),
         'subscription': {},
+        'grant_has_no_token': True if grant.token_address == '0x0000000000000000000000000000000000000000' else False,
```

Comment:
@captnseagraves Somewhat similar to what we discussed about the `not` reserved word, you can do something like:

`'grant_has_no_token': grant.token == '0x0000000000000000000000000000000000000000'`